### PR TITLE
add cloud command reference for v0.6.0

### PR DESCRIPTION
**Pull request by bacongobbler**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title`, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

✅ content/cloud/cloud-command-reference.md

- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors
- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/cloud/cloud-command-reference.md [hunk 1]
```diff
@@ -1608,6 +1887,48 @@ OPTIONS:
 
 {{ blockEnd }}
 
+{{ startTab "v0.6.0"}}
```
Comment (itowlson) [inline content/cloud/cloud-command-reference.md:1890]: I think there should be a `logout` command between `login` and `logs` (unexciting though it is).

Comment (tpmccallum) [reply content/cloud/cloud-command-reference.md:1917]: Thanks @itowlson - good spotting

**Comment by tpmccallum** [comment]
Thanks so much for this @bacongobbler; greatly appreciated.
I have just added a couple of tweaks and am happy to approve & merge.

**Review by tpmccallum — APPROVED** [review]
LGTM @bacongobbler 
Thanks so much!

**Comment by tpmccallum** [comment]
Build error unrelated.
Cloud CLI content LGTM - merging.

**Pull request merged**